package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "dev": "LOG_LEVEL=trace NODE_ENV=development nodemon --exec 'npm start'",
     "start": "probot run ./index.js",
-    "test": "jest && standard",
+    "test": "jest __tests__/unit/validators/options_processor/options/or.test.js && standard",
```

Comment:
Please fix the above. It looks like it's an unintentional commit. May I recommend doing an `alias` for shortcuts in your shell instead of modifying this file?

Reply:
Sorry, I'll try using `alias`